src/test/resources/features/actions/line_mover.feature

```diff
@@ -1,44 +1,59 @@
 Feature: Line mover
-  Scenario Outline: Move rules up
+  Scenario Outline: Permutation up/down between rules sections
```

Comment:
A permutation is a math term, means `a way, especially one of several possible variations, in which a set or number of things can be ordered or arranged.`. Please always check your translation in English-English dictionary. So here it sounds confusing. Please use here a more accurate translation, e.g. move up/down sections, swap rule like sections, etc

Reply:
P.S: I renamed tests